src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1503,30 +1501,26 @@ private String[] getCommandTokens(String command) {
         return commandTokens;
     }
 
-    private void stopContainer() {
+    private void stopContainer()  throws PluginExecutionException {
         try {
             serverFullyStarted.set(false);
 
-            // see if docker run command (container) is still running before trying to stop it.
-            if (dockerRunProcess != null && dockerRunProcess.isAlive()) {
+            // see if container run command (container) is still running before trying to stop it.
+            if (containerRunProcess != null && containerRunProcess.isAlive()) {
                 info("Stopping container...");
-                String dockerStopCmd = "docker stop " + containerName;
+                String containerStopCmd = getContainerCommandPrefix() + "stop " + containerName;
                 debug("Stopping container " + containerName);
-                execDockerCmd(dockerStopCmd, DOCKER_TIMEOUT + 20); // allow extra time for server shutdown
+                execContainerCmd(containerStopCmd, DOCKER_TIMEOUT + 20); // allow extra time for server shutdown
                 writeDevcMetadata(false);
             }
         } catch (RuntimeException r) {
             error("Error stopping container: " + r.getMessage());
             throw r;
         } finally {
-            dockerRunProcess = null;
+            containerRunProcess = null;
         }
     }
 
-    private String execDockerCmd(String command, int timeout) {
-        return execDockerCmd(command, timeout, true);
```

Comment:
See this used to pass `true` for the boolean that controls whether exceptions get thrown, but when you moved this method to `AbstractContainerSupportUtil` you changed it to `false`.